README.md

```diff
@@ -163,11 +163,12 @@ A VM can be tuned at two different places with the following keys:
 - `distro`: the name of the base distro image to use, it's the only mandatory parameter.
 - `name`: the VM name
 - `memory`: the amount of memory to dedicate to the VM
-- `root_disk_size`: the size of the root disk in GB
```

Comment:
Can you please keep the parameter in the documentation, just clarify it's deprecated.